documentation/whats-new.md

```diff
@@ -1,10 +1,18 @@
 <!--
- Copyright 2020 Google LLC
+ Copyright 2020-2021 Google LLC
```

Comment:
All files that have been changed should have their copyright header updated. You can use this as an example for how to do it.

Reply:
Updated